includes/Checker/Checks.php

```diff
@@ -0,0 +1,125 @@
+<?php
+/**
+ * Class WordPress\Plugin_Check\Checker\Checks
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\Checker;
+
+use WordPress\Plugin_Check\Checker\Check_Context;
+use ReflectionClass;
+use Exception;
+
+/**
+ * Class to run checks on a plugin.
+ *
+ * @since n.e.x.t
+ */
+class Checks {
+
+	/**
+	 * Context for the plugin to check.
+	 *
+	 * @since n.e.x.t
+	 * @var Check_Context
+	 */
+	protected $check_context;
+
+	/**
+	 * Sets the main context and the main file of the plugin to check.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param string $plugin_main_file Absolute path to the plugin main file.
+	 */
+	public function __construct( $plugin_main_file ) {
+		$this->check_context = new Check_Context( $plugin_main_file );
+	}
+
+	/**
+	 * Runs a single check against the plugin.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param array $checks Ab array of Check class names to run.
```

Comment:
See other comment, should probably be `An array of Check instances to run.` (also see typo in the first word).